tools/common/lib.go

```diff
@@ -73,27 +73,57 @@ const (
 	ConsumerGroupType = "CG"
 	// MinUnconsumedMessagesRetentionForMultiZoneDest is the minimum unconsumed retention allowed
 	MinUnconsumedMessagesRetentionForMultiZoneDest = 3 * 24 * 3600
+	// MinConsumedMessagesRetention is the minimum consumed retention
+	MinConsumedMessagesRetention = 60
+	// MaxConsumedMessagesRetention is the maximum consumed retention
+	MaxConsumedMessagesRetention = 100 * 24 * 3600
+	// MinUnconsumedMessagesRetention is the minimum unconsumed retention
+	MinUnconsumedMessagesRetention = 60
+	// MaxUnconsumedMessagesRetention is the maximum unconsumed retention
+	MaxUnconsumedMessagesRetention = 100 * 24 * 3600
+	// MinLockTimeoutSeconds is the minimum lock timeout seconds
+	MinLockTimeoutSeconds = 0
+	// MaxLockTimeoutSeconds is the maximum lock timeout seconds
+	MaxLockTimeoutSeconds = 100 * 24 * 3600
+	// MinMaxDeliveryCount is the minimum value for max delivery count
+	MinMaxDeliveryCount = 0
+	// MaxMaxDeliveryCount is the maximum value for max delivery count
+	MaxMaxDeliveryCount = math.MaxInt32
```

Comment:
we need a cap. Maybe look at all cgs on prod, and find out what's the max we're using today